src/templates/next/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,41 @@
+import { BiChevronDown } from "react-icons/bi"
+import { AccordionProps } from "~/common"
+import BaseParagraph from "../shared/Paragraph"
+import { Paragraph } from "../../typography/Paragraph"
+import Image from "../Image"
+import OrderedList from "../OrderedList"
+import UnorderedList from "../UnorderedList"
+
+const Accordion = ({ summary, details }: AccordionProps) => {
+  return (
+    <details className="[&_>_summary]:open:font-semibold [&_>_summary_>_svg]:open:rotate-180 border-y has-[+_details]:border-b-0 border-divider-medium">
+      <summary className="my-2 px-4 py-3 flex flex-row gap-3 outline-none list-none focus:ring-2 focus:ring-offset-0 focus:ring-focus-outline">
```

Comment:
needs text colour here

Reply:
Added in beeae4b90aa4964cbd4a2d1dbe6ec92c19ac8e56.